packages/deploy/deploy/500_marketplace/503_royalty_registry_set_whitelist_roles.ts

```diff
@@ -0,0 +1,68 @@
+import {DeployFunction} from 'hardhat-deploy/types';
```

Comment:
can you use the same naming than in assets ?
405_asset_create_setup.ts
503_royalties_registry_setup.ts

and btw it's royalties_registry not royalty_registry